x-pack/plugins/security_solution/server/lib/detection_engine/rules/utils.ts

```diff
@@ -102,7 +107,9 @@ export const transformFromAlertThrottle = (
   legacyRuleActions: LegacyRuleActions | null | undefined
 ): string => {
   if (legacyRuleActions == null || (rule.actions != null && rule.actions.length > 0)) {
-    if (rule.muteAll || rule.actions.length === 0) {
+    const noResponseActions = !(rule.params as { responseActions: RuleResponseAction[] })
+      ?.responseActions?.length;
+    if (rule.muteAll || (rule.actions.length === 0 && noResponseActions)) {
```

Comment:
I have the same question here as in the above comment.

We will need to test it with bulk editing rule actions.

Reply:
Remove the connection between throttle and respponseActions 👍